server/conf/evolutions/default/81.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE programs ADD COLUMN IF NOT EXISTS localized_short_description jsonb DEFAULT ('{"isRequired": true, "translations": {"en_US": ""}}')::jsonb;
```

Comment:
**suggestion (non-blocking):** We could check for existence of the table, just to be extra cautious.  `ALTER TABLE if exists programs`.  And same in the downs.